### DISCUSSION TITLE
Brew doesn't call in Bash

### DISCUSSION THREAD

**Discussion by AlexsanderCallou** [Homebrew/discussions#3370] [Linux]
Hi, after installing Linux Brew in popOs 22.04, following the guideline to have Brew on Bash.

At the end of my .profile, I have the following lines:

#set PATH so it includes user's private bin if it exists
if [ -d "$HOME/.local/bin" ] ; then
    PATH="$HOME/.local/bin:$PATH"
fi    

eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"

export PATH="/home/linuxbrew/.linuxbrew/bin:$PATH"

But still, don't have Brew in bash, can someone help me?

Ps: I dont have .profile_login or .bash_profile in my system.

after install did the steps:

`echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/alex/.profile
`
`eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
`

and 
`test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile
echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.profile`

**Comment by SMillerDev** [comment]
The shellenv line probably won't work if you also add it to the path manually

**Comment by gromgit** [comment]
What's the output of the following?
```
echo $PATH
/home/linuxbrew/.linuxbrew/bin/brew config
/home/linuxbrew/.linuxbrew/bin/brew doctor
```

**Comment by AlexsanderCallou** [comment]
Don't really sure what was the issue, but when a boot the computer this morning and did the steps again, now its working!

Thanks!
